configure.ac

```diff
@@ -146,13 +146,19 @@ AC_SEARCH_LIBS([gethostbyname], [nsl],,[AC_MSG_ERROR([cannot find gethostbyname(
 AC_SUBST([nsl_LIBS],[$LIBS])
 AX_RESTORE_FLAGS
 
+USE_JOURNAL="no"
+PKG_CHECK_MODULES([SYSTEMD], [libsystemd], [have_systemd=yes],[have_systemd=no])
 if test "x$enable_systemd_journal" = "xyes" ; then
-  AX_SAVE_FLAGS
-  AC_SEARCH_LIBS([sd_journal_send], [systemd-journal],,[AC_MSG_ERROR([cannot find sd_journal_send() function])])
-  AC_SUBST([systemd_LIBS], [$LIBS])
-  AX_RESTORE_FLAGS
+  if test "x$have_systemd" = "xyes" ; then
+     AC_DEFINE_UNQUOTED(USE_JOURNAL, 1, [Use systemd journal logging])
+     USE_JOURNAL="yes"
+     echo "CC: found systemd & journal requested"
```

Comment:
This debug print should go or be replaced by something without CC: prefix :)

Reply:
err yes. That should never have been left in. TBH I only pushed it to move a copy to my Fedora system for testing!